package.json

```diff
@@ -53,8 +53,9 @@
     "codegen:cleanup": "rm --force --verbose ./openapitools.json",
     "codegen:lerna": "lerna run codegen",
     "codegen:warmup-mkdir": "make-dir ./node_modules/@openapitools/openapi-generator-cli/versions/",
-    "codegen:warmup-v6.6.0": "nwget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/6.6.0/openapi-generator-cli-6.6.0.jar -O ./node_modules/@openapitools/openapi-generator-cli/versions/6.6.0.jar",
-    "watch-other": "lerna run --parallel watch",
+    "codegen:warmup-v5.2.1": "nwget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.2.1/openapi-generator-cli-5.2.1.jar -O ./node_modules/@openapitools/openapi-generator-cli/versions/5.2.1.jar",
```

Comment:
The removed changes were the correct ones

Reply:
reverted.